x-pack/test_serverless/api_integration/test_suites/common/telemetry/telemetry_config.ts

```diff
@@ -15,58 +15,59 @@ export default function telemetryConfigTest({ getService }: FtrProviderContext)
   const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   describe('/api/telemetry/v2/config API Telemetry config', function () {
-    const baseConfig = {
-      allowChangingOptInStatus: false,
-      optIn: true,
-      sendUsageFrom: 'server',
-      telemetryNotifyUserAboutOptInDefault: false,
-      labels: {
-        serverless: 'observability',
-      },
-    };
     let roleAuthc: RoleCredentials;
-    let internalReqHeader: InternalRequestHeader;
 
     before(async () => {
       roleAuthc = await svlUserManager.createApiKeyForRole('admin');
-      internalReqHeader = svlCommonApi.getInternalRequestHeader();
     });
 
     after(async () => {
       await svlUserManager.invalidateApiKeyForRole(roleAuthc);
     });
 
+    const baseConfig = {
+      allowChangingOptInStatus: false,
+      optIn: true,
+      sendUsageFrom: 'server',
+      telemetryNotifyUserAboutOptInDefault: false,
+    };
+
     it('GET should get the default config', async () => {
       const { body } = await supertestWithoutAuth
         .get('/api/telemetry/v2/config')
-        .set(internalReqHeader)
+        .set(svlCommonApi.getCommonRequestHeader())
         .set(roleAuthc.apiKeyHeader)
         .expect(200);
+
       expect(body).toMatchObject(baseConfig);
     });
 
-    it.skip('GET should get updated labels after dynamically updating them', async () => {
+    it('GET should get updated labels after dynamically updating them', async () => {
+      const { body: initialConfig } = await supertestWithoutAuth
+        .get('/api/telemetry/v2/config')
+        .set(svlCommonApi.getCommonRequestHeader())
+        .set(roleAuthc.apiKeyHeader)
+        .expect(200);
```

Comment:
This suite is now in the `common` dir, so we need to be more flexible with the expected config.

This test is validating that changing a `telemetry.label` through `PUT /internal/core/_settings`, updates that label, without affecting the rest of the document.